docs/UserGuide.md

```diff
@@ -58,121 +68,131 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 </div>
 
-Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+Example: `project n/Momentum d/CS2103T Team Project t/impt`
+
+Result: Creates a project named “Momentum” with a description “CS2103T Team Project” and a tag "impt".
+
+### Updating a project: `edit`
+
+Update a project that has been previously created.
+
+Format: `edit PROJECT_ID n/NAME d/DESCRIPTION [t/TAG]`
+
+* Edits the person at the specified `PROJECT_ID`.
+* The id refers to the id number shown in the displayed project list.
+* The id **must be a positive integer** 1, 2, 3, …​
+
+Example: `project 3 n/NewMomentum d/NewDescription t\normal`
+
+Result: Updates the project with id 3. The name will be changed to “NewMomentum”, the description will be changed to “NewDescription”, the tag will be changed to normal.
+
+### Deleting a project: `delete`
+
+Deletes a project in the list.
+
+Format: `delete PROJECT_ID`
+
+* Deletes the person at the specified `PROJECT_ID`.
+* The id refers to the id number shown in the displayed project list.
+* The id **must be a positive integer** 1, 2, 3, …​
 
-### Listing all persons : `list`
+Example: `delete 2`
 
-Shows a list of all persons in the address book.
+Result: Deletes the second project in the list.
 
-Format: `list`
+### View
 
-### Editing a person : `edit`
+View different tabs of the application.
 
-Edits an existing person in the address book.
+Format: <br> `\home` <br> `\stats` <br> `\projects` <br> `\settings`
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+* /home brings user to home dashboard
+* /stats brings user to Statistics tab
+* /projects brings user to Projects tab
+* /settings brings user to Settings tab
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+Result: Moves to different tabs in the application.
 
-Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+#### Home View
 
-### Locating persons by name: `find`
+<img src="images/user-guide/homeView.ppg" width="300" />
 
-Finds persons whose names contain any of the given keywords.
+#### Projects View
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+<img src="images/user-guide/projectsView.png" width="300" />
 
```

Comment:
Should we replace these images with @khoodehui 's mockups? At least the ones that are ready

Reply:
Yeahh we should